### PR TITLE
[bug 1159045] Add GA tracking to instant search

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: kitsune/sumo/static/sumo/js/instant_search.js [hunk 1]
```diff
@@ -93,6 +101,8 @@
       });
     }
 
+    _gaq.push(['_trackEvent', 'Instant Search', 'Search', $this.data('href')]);
```
Comment (mythmon) [inline kitsune/sumo/static/sumo/js/instant_search.js]: We need to track these as page views that match Google Analytics' search tracking so we can use all the search metrics.

Comment (mythmon) [inline kitsune/sumo/static/sumo/js/instant_search.js]: There is some information about this here: https://support.google.com/analytics/answer/1012264?hl=en#Post

In particular, the event should be something like `ga('send', 'pageview', '/search?q=keyword');`

**Comment by rehandalal** [comment]
updated ^

**Comment by mythmon** [comment]
I think this will work. It can't be worse than what we have now.

r+, though I can't merge it (looks like merge conflict?). Want to fix this up and land it?

**Pull request merged**